src/main/resources/application.yaml

```diff
@@ -69,7 +69,7 @@ quarkus:
 
   liquibase:
     migrate-at-start: true
-    change-log: db/changeLog.yaml
+    change-log: db/damap/changeLog-root.yaml
```

Comment:
Shall the namespacing be more explicit to be safe? `at/ac/tuwien/damap` (as is done with java source files)

Reply:
Would it be necessary in that case? I am not familiar with the reason why it is done to be honest. Is it to prevent conflicts with dependencies? Maybe you could give me an example?

Reply:
The namespacing with the URL makes it very explicit where something originates from in order to prevent conflicts or misunderstandings.
Imagine TU Graz, TU Wien and the open source DAMAP project creating source files under the `damap` namespace only. This would result in conflicts, if two files are named the same.

DAMAP: `damap/ProjectService.java`
TU Wien: `damap/ProjectService.java`
TU Graz: `damap/ProjectService.java`

Now, which one should be used when `damap/ProjectService.java` is included?
Additionally, other dependencies might provide a similar file structure, which would also lead to the same issue.

Maybe we could also namespace everything under the resources folder?
```
src
  - main
    - resources
      - at
        - ac
          - tuwien
            - damap
              - db
              - schema
              - spec
              - templates
        - tugraz
          - damap
              - db
              - schema
              - spec
              - templates
```

Up for discussion though :)

Reply:
I see where you are coming from. But I am not really familiar with how the institutional backends look. Do they have the same file structure as the damap project? If yes, then it should not be possible to create a file/folder with a duplicate path. Or is it just about being clear where the files originated? Maybe you should talk with @ZenoLC  about this, to find the best possible solution.

Reply:
It would be great to be able to create whatever folder structure a custom project wants to setup, without having to care about another one. If a custom project adds a `damap/projects.yaml`, which does not exist in the open source project at the time, everything works fine. Down the line, the open source project also adds a `damap/projects.yaml` file. Now the custom project does not build anymore and has to be adapted. This can be mitigated in the first place via namespacing.

Reply:
Which solution would you suggest? Should the namespacing start at db or at resources?
Also, what namespace should we use for the open source? db/at/ac/tuwien/damap would clash with the tuwien custom project.

Reply:
Putting it at resources ensures that it only has to be setup once per project and not repeated per folder. Not sure about it though, since I do not have that much experience with java projects.
Maybe `org.damap`? As said, not sure about the decision at all. Up for discussion with @ZenoLC as well (especially for the custom project)

Reply:
As discussed, I have namespaced the resources folder. The many changes are due to changing file locations / filepaths. Notice that I added a logicalFilePath attribute to changeLog.yaml. 
`- logicalFilePath:  db/changeLog.yaml`
This was necessary due to the way liquibase distinguishes between changesets, namely with id/author/filepath. The hash is used to detect if a changeset changed, but id/author/filepath determines the changeset identity. logicalFilePath makes it so that liquibase uses the path specified in the logicalFilePath attribute to ascertain identity - the physical file path stays as is (even in log files).